src/views/Records/Records.vue

```diff
@@ -72,10 +71,11 @@ import JumpToTop from "@/components/Navigation/jumpToTop";
 import recordsLabels from "@/data/recordsTypes.json"
 import FilterChips from "@/components/Records/Search/Header/FilterChips";
 import filterChipsUtils from "@/utils/filterChipsUtils";
+import Footer from "@/components/Navigation/Footer";
 
 export default {
   name: "Records",
-  components: { JumpToTop, SearchOutput, SearchInput, FilterChips},
+  components: {Footer, JumpToTop, SearchOutput, SearchInput, FilterChips},
```

Comment:
Why not handle the footer from App.vue by removing the v-if in the footer?

Reply:
You are right we can do it. its been done in the next branches. so this will be overwritten..